Source/ASVideoPlayerNode.mm

```diff
@@ -284,15 +282,23 @@ - (void)createControls
 
     if (_delegateFlags.delegateCustomControls && _delegateFlags.delegateLayoutSpecForControls) {
       NSDictionary *customControls = [_delegate videoPlayerNodeCustomControls:self];
```

Comment:
In general we should avoid calling delegates with the lock held. May not worth address in this PR though.